thorlcr/slave/slavmain.cpp

```diff
@@ -768,9 +770,9 @@ class CKJService : public CSimpleInterfaceOf<IKJService>, implements IThreaded,
                     if (last || (replyMb.length() >= DEFAULT_KEYLOOKUP_MAXREPLYSZ))
                     {
                         countMarker.write(rowNum-rowStart);
-                        replyMb.append(kmc->queryKeyManager()->querySeeks()-startSeeks);
-                        replyMb.append(kmc->queryKeyManager()->queryScans()-startScans);
-                        replyMb.append(kmc->queryKeyManager()->queryWildSeeks()-startWildSeeks);
+                        replyMb.append(stats.getStatisticValue(StNumIndexSeeks)-startSeeks);
+                        replyMb.append(stats.getStatisticValue(StNumIndexScans)-startScans);
+                        replyMb.append(stats.getStatisticValue(StNumIndexWildSeeks)-startWildSeeks);
```

Comment:
see previous comment (but could tackle as separate issue).

Reply:
I'll do this as separate issue: https://track.hpccsystems.com/browse/HPCC-29822